fuel/src/test/kotlin/com/github/kittinunf/fuel/MockHttpTestCase.kt

```diff
@@ -1,20 +1,39 @@
 package com.github.kittinunf.fuel
 
+import com.github.kittinunf.fuel.core.FuelManager
+import com.github.kittinunf.fuel.core.Method
+import com.github.kittinunf.fuel.core.Request
 import org.junit.After
 import org.junit.Before
+import org.slf4j.event.Level
 
 abstract class MockHttpTestCase : BaseTestCase() {
 
     protected lateinit var mock: MockHelper
 
     @Before
     fun setup() {
+        // You can set the log level to INFO or TRACE to see all the mocking logging
         this.mock = MockHelper()
-        this.mock.setup()
+        this.mock.setup(Level.INFO)
     }
 
     @After
     fun tearDown() {
         this.mock.tearDown()
     }
+
+    fun reflectedRequest(
+        method: Method,
+        path: String,
+        parameters: List<Pair<String, Any?>>? = null,
```

Comment:
[Nits] Use type alias that we defined earlier? `Parameters`

Reply:
Definitely!